config/routes.rb

```diff
@@ -68,6 +68,8 @@
     root 'organisation_user#index', as: :authenticated_organisation_user_root
   end
 
+  resources :organisations
```

Comment:
This creates routes for all CRUD actions - we only want `index`.

```suggestion
  resources :organisations, only: [:index]
```